site/templates/meet.json.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @var Kirby\Cms\Pages $upcoming
+ * @var Kirby\Cms\Pages $past
+ */
+$upcomingEvents = [];
+$pastEvents     = [];
+
+$since = param('since');
+
+foreach ($upcoming as $event) {
+	$upcomingEvents[] = [
+		'title'    => $event->shortTitle()->value(),
+		'date'     => $event->datetime()->format('c'),
+		'timezone' => $event->timezone()->value(),
+		'link'     => $event->link()->value(),
+		'isMeetup' => $event->isMeetup(),
+	
+	];
+}
```

Comment:
`Suggestion`: This could be shortened with:

```suggestion
$upcomingEvents = $upcoming->toArray(fn ($event) => [
	'title'    => $event->shortTitle()->value(),
	'date'     => $event->datetime()->format('c'),
	'timezone' => $event->timezone()->value(),
	'link'     => $event->link()->value(),
	'isMeetup' => $event->isMeetup(),
]);
```

Reply:
Or `->values(fn () => ...)` if you don't want keys.